src/cpp/gov/nist/ucef/hla/FederateAmbassador.cpp

```diff
@@ -30,7 +30,7 @@ namespace ucef
 	                                                            throw( FederateInternalError )
 	{
 		SynchPoint synPoint = ConversionHelper::StringToSynchPoint( label );
```

Comment:
Minor thing, but maybe this should be `syncPoint` (with a 'c'):
```suggestion
		SynchPoint syncPoint = ConversionHelper::StringToSynchPoint( label );
```